README.md

```diff
@@ -75,15 +75,24 @@ The above `nginx` configuration proxies `https://localhost:4437` to `http://loca
 
 ### Start emulator
 
-```
+```bash
 npm run dev
 ```
 
 ### Shared storage
 
-To run this demo, [follow the instructions](https://developer.chrome.com/docs/privacy-sandbox/shared-storage/#try-the-shared-storage-api) for enabling the Privacy Sandbox Ads APIs experiment flag at **chrome://flags/#privacy-sandbox-ads-apis**.
+To run this demo, enable the experiment flags:
+
+* chrome://flags/#enable-fenced-frames
```

Comment:
Hmm. I have the FF flag set as "Default" and it works properly. Would you double check that you need the FF flags to be changed for the demo to run?

Reply:
Oh cool, it does work fine without this flag. Is it possible the other flag also enables fenced frames?

Reply:
Yea, that's possible but I'm not 100% sure.